src/Exporter.php

```diff
@@ -707,9 +707,17 @@ protected function getRequiredVariationValues():array
             $variationValues[] = 'variationAttributeValues';
         }
 
+        $itemProperties = $this->registry->get('itemproperties');
```

Comment:
Add comments as it's not clear what itemproperties and properties are. Weird naming from Plenty